src/charm.py

```diff
@@ -134,6 +138,7 @@ def _update_new_metrics_consumer(self, event):
     def _update_all_metrics_consumers(self, _):
         """Update all scrape configuration jobs for all metrics consumers."""
         if not self.unit.is_leader():
+            self.unit.status = WaitingStatus("follower unit")
```

Comment:
Is it consistent with the rest of charm that status is set by non-hook methods / methods other than `_configure`?

Reply:
The message could probably be clearer than "follower unit". I'm not sure what a good tradeoff between brevity and communication is. Maybe `WaitingStatus("Not the leader. Not providing endpoints to Prometheus")` or similar

Reply:
Brevity may be a good idea in a multi-column text console. Having said that it is not some thing I want to belabour beyond a point. I can add a line to the README.md saying that there are no functional gains by scaling this charm since "follower units" (non-leader) do not forward any information to the MetricsEndpointConsumer.

ATTN: @mmanciop

Reply:
The gist was more that "follower unit" may not mean anything to the Juju admin or indicate what the unit is or is not doing. We should provide a message which tells them and does not require looking up a README. It should be immediately clear.

Reply:
Would "unused unit" work instead of "follower unit" ?

Reply:
Up to @mmanciop, but that makes more sense to me.

Reply:
I like the brevity of “follower unit”, and I think I asked Balbir for that, But Ryan makes an excellent point in saying that the message does not explain what it does for the user. What about “follower unit, inactive until is leader”

Reply:
Changed to "unused unit" since that is what every one suggested today. I have no problem changing it again to "follower unit, inactive until is leader" though to be strictly grammatically correct it ought to be "follower unit, inactive until it is leader".

Reply:
Let’s cut this short and go with “inactive unit”.